src/common/resolve.rs

```diff
@@ -71,6 +72,29 @@ impl CurrentUser {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct AuthUser(User);
+
+impl AuthUser {
+    pub fn from_current_user(user: CurrentUser) -> Self {
+        Self(user.inner)
+    }
+
+    pub fn resolve_root_for_rootpw() -> Result<Self, Error> {
+        Ok(Self(
+            User::from_uid(UserId::ROOT)?.ok_or(Error::UserNotFound("current user".to_string()))?,
```

Comment:
Shouldn't this be:
```suggestion
            User::from_uid(UserId::ROOT)?.ok_or(Error::UserNotFound("root".to_string()))?,
```
?

Reply:
Yes